src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(images/star.svg) center no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
please make sure that this works right

Reply:
My problem was that I didn't deploy changes. I thought it was an automatic process.